examples/zookeeper/README.md

```diff
@@ -0,0 +1,63 @@
+Replicated Zookeeper Template
+=============================
+
+ZooKeeper is a distributed, open-source coordination service for distributed
+applications. It exposes a simple set of primitives that distributed
+applications can build upon to implement higher level services for
+synchronization, configuration maintenance, and groups and naming. It is
+designed to be easy to program to, and uses a data model styled after the
+familiar directory tree structure of file systems. It runs in Java and has
+bindings for both Java and C.
+
+For more informations about Zookeeper visit:
+
+http://zookeeper.apache.org/doc/r3.4.6/zookeeperOver.html
+
+### Pre-requires
+
+You can 'pre-pull' the Docker image used by this template by:
+
+```
+$ docker pull mfojtik/zookeeper-346-jdk7
```

Comment:
@bparees jdk is needed for zookeeper, I thought we are referring to the base image, in this case it is jdk7 (the base for that is 'jboss/base' and the base for that is 'fedora:20' ;-) To base this on centos, I would have to install all the jdk7 bloat into it and I'm not fan of doing that, especially if we case use our own jdk7 image from jboss guys ;-)